src/user/contributing.rst

```diff
@@ -49,4 +49,20 @@ We are glad to know that you would like to contribute to the ``conda-forge`` doc
 5. Submit a `pull request
    <https://help.github.com/articles/about-pull-requests/>`_ to the main repository proposing your changes.
 
+
+   Writing guidelines:
+
+Some basic writing guidelines should be kept in mind before you start contributing.
```

Comment:
Maybe add a colon to the end of this sentence instead of a period, since you have a list of guidelines below it.

Also, I'd recommend a blank line between lines 55 and 56.